### PR TITLE
Adding more keyboard navigation shortcuts.

### LINKED ISSUE #161: [Suggestion] Add keyboard shortcuts for copying entries
## This issue is a 

- [ ] bug
- [x] feature
- [ ] question

### Please describe the current behavior, and explain why it's bad.

I cannot copy the password using keyboard. I have to use my mouse to do so.

### Please describe how you think it should change.

Please add support for using keyboard shortcut for copying the password (e.g. `Ctrl+C`). I want to be able to copy the username as well (e.g. `Ctrl+B`).

### PR DESCRIPTION
**Pull request by zmilonas**
Adding password copy and username copy as well as up arrow list navigation.

The only problem with Ctrl+B is that it opens the bookmarks in Firefox.

Would also be usable to write tests for keyboard navigation.

Fixes #161

### CODE REVIEW

**Comment by subdavis** [comment]
I think these should also be implemented on the entry details page, if they are going to be added.  

It would also be good to add a key for copying the OTP token if it exists.  

If you would like me to make those chances, please lmk.

**Comment by zmilonas** [comment]
Ok thank you :)
What should we do about Cmd-B being a bookmark shortcut in Firefox? Do you have a suggestion for a better shortcut?

**Comment by subdavis** [comment]
I think all you need to do is `event.preventDefault()` after handling cmd+b

This should stop propagation on the key event and Firefox won't handle it.

**Comment by zmilonas** [comment]
I actually tried both this and `event.stopPropagation()` and they seem not to have any effect. :/

**Comment by zmilonas** [comment]
I can't actually find good 2 key shortcuts for firefox and with your suggestions there are 3 shortcuts to be made: 
- Copy Username
- Copy Password
- Copy OTP

File: services/unlockedState.js [hunk 1]
```diff
@@ -117,15 +118,20 @@ function UnlockedState($router, keepassReference, protectedMemory, settings) {
 	}
 
 	//get clear-text password from entry
-	function getPassword(entry) {
-		return my.getDecryptedAttribute(entry, 'password');
+	function getAttribute(entry, attr = 'password') {
```
Comment (subdavis) [inline services/unlockedState.js]: I like how you did this.

**Pull request merged**